src/components/TextInput/index.js

```diff
@@ -105,6 +105,7 @@ class TextInput extends Element {
     _onInputFocus(evt) {
         this.class.add(pcuiClass.FOCUS);
         this.emit('focus', evt);
+        this._prevValue = this.value;
```

Comment:
might want to declare/create this member in the constructor?

Reply:
Will do 👍